frontend/app/src/types/history/tx.ts

```diff
@@ -83,6 +83,11 @@ export interface AddressesAndEvmChainPayload {
   readonly evmChain: string;
 }
 
+export interface AddressAndEvmChainPayload {
```

Comment:
I am pretty sure I already added a type that takes address and evmChain in the websocket PR, not sure if I exported it though. Maybe it would make sense to use that one?